tests/timeseries/mock_resampler.py

```diff
@@ -222,15 +222,18 @@ async def _handle_resampling_requests(self) -> None:
             name = request.get_channel_name()
             if name in self._forward_tasks:
                 continue
-            basic_recv_name = f"{request.component_id}:{request.metric_id}"
-            recv = self._basic_receivers[basic_recv_name].pop()
-            assert recv is not None
+            input_chan_recv_name = f"{request.component_id}:{request.metric_id}"
+            input_chan_recv = self._input_channels_receivers[input_chan_recv_name].pop()
+            assert input_chan_recv is not None
+            output_chan_sender: Sender[Sample[Quantity]] = (
+                self._channel_registry.get_or_create(
+                    Sample[Quantity], name
+                ).new_sender()
+            )
```

Comment:
Interesting, this is better than `basic_receivers`, but `input_receivers` and `output_sender` might have even nicer.  `input_channels_receivers` make me think that they receive channels and not input values.

Reply:
IIRC we wanted to mainly get rid of `basic` in the variable name because it was confusing/difficult-to-understand what a `basic_receiver` was. I do agree having only `receivers` and `sender` in the name would have suffice to understand they are channels given the context. However I don't think it causes any misunderstanding if we keep `chan/channels` explicitly in the variable names. I mean I haven't seen yet any scenarios where we send/receive channels